src/fontra/views/editor/panel-reference-font.js

```diff
@@ -264,15 +287,115 @@ export default class ReferenceFontPanel extends Panel {
 
     this.controller.addKeyListener("charOverride", (event) => {
       this.editorController.canvasController.requestUpdate();
+      this.requestReferenceFontsPreview();
     });
 
     this.controller.addKeyListener("languageCode", (event) => {
       this.editorController.canvasController.setLangAttribute(this.model.languageCode);
+      this.requestReferenceFontsPreview();
     });
 
     this.editorController.canvasController.setLangAttribute(this.model.languageCode);
 
     referenceFontModel = this.model;
+
+    this.editorController.sceneSettingsController.addKeyListener(
+      ["selectedGlyph", "glyphLines"],
```

Comment:
I think it's enough to listen to "selectedGlyphName". Can you try that?

Reply:
Yes, it works